xslt/Pixels.xsl

```diff
@@ -167,27 +168,31 @@
         <xsl:param name="idx" />
         <xsl:element name="ome:Pixels">
             <xsl:attribute name="ID">
-                <xsl:value-of select="concat('Pixels:', position(), '-', $idx)"/>
+                <xsl:value-of select="concat('Pixels:', position() - 1, '-', $idx)"/>
```

Comment:
When working on this, I noticed that the index for `Pixels` started at 1 instead of 0 like all the other elements. This change fixes that.